starter-code/javascript/chronometer.js

```diff
@@ -1,40 +1,59 @@
 // Constructor
-// function Chronometer() {
-
-// }
-
-// Chronometer.prototype.startClick = function () {
-
-// };
-
-// Chronometer.prototype.setMinutes = function () {
-  
-// };
-
-// Chronometer.prototype.setSeconds = function () {
-  
-// };
-
-// Chronometer.prototype.twoDigitsNumber = function () {
-  
-// };
-
-// Chronometer.prototype.setTime = function () {
-
-// };
+function Chronometer() {
+  this.currentTime = 0;
+  this.intervalId = 0;
+
+}
+
+Chronometer.prototype.startClick = function () {
+  var self = this 
+  setInterval(function() {
+    self.currentTime += 1;
+    self.setTime();
+  }, 1000);
+};
+
+Chronometer.prototype.setMinutes = function () {
+  if (this.currentTime < 60) {
+    return 0;
+  } else {
+    return Math.floor(this.currentTime / 60);
```

Comment:
```return Math.floor(this.currentTime / 60);``` will return 0 if currentTime is less than 60. You don't need the first condition